docker-compose.yml

```
@@ -1,6 +1,28 @@
 version: '3.8'
 
 services:
+  postgres:
+   image: postgres:15-alpine
```

yamllint (1.35.1)

[warning] wrong indentation: expected 4 but found 3

(indentation)